data/pigui/modules/station-view/04-shipMarket.lua

```diff
@@ -195,8 +195,8 @@ local tradeMenu = function()
 				ui.sameLine()
 				manufacturerIcon(selectedItem.def.manufacturer)
 				local shipBought = false
-				ui.withFont(pionillium.medlarge.name, orbiteer.medlarge.size, function()
-					shipBought = ui.coloredSelectedButton(l.BUY_SHIP, widgetSizes.buyButton, false, colors.buttonBlue, nil, true)
+				ui.withFont(pionillium.medlarge.name, pionillium.medlarge.size, function()
+					shipBought = ui.button(l.BUY_SHIP, widgetSizes.buyButton)
```

Comment:
When you come across a withFont call like this, please collapse the separate `.name` and `.size` parameters into a single parameter passing the font object directly.

Reply:
Done!